fenix/app/.experimenter.yaml

```diff
@@ -15,6 +15,14 @@ extensions-process:
     enabled:
       type: boolean
       description: "If true, the extensions process is enabled."
+fx-suggest:
+  description: A feature that provides Firefox Suggest search suggestions.
+  hasExposure: true
```

Comment:
I'm not sure, actually! I admit I didn't know what an [exposure](https://experimenter.info/telemetry#exposure) was until you called it out here—I think `./gradlew app:assembleDebug` regenerated this file for me after I added the feature to `nimbus.fml.yaml`, and I committed it without thinking about it too hard.

Reply:
Reading that doc a bit more, though, I don't think exposures are the right tool for us, and so maybe this ought to be `false`?

I know we'll want to record impressions and clicks on Firefox Suggest suggestions, but for sponsored ones only, and pass along special impression and click URLs as extra keys. I wonder if having the component emit those facts—plus new facts for `fx_suggest_suggestion_clicked`, matching the existing `{provider}_suggestion_clicked` ones we have today—would be more useful.

WDYT?

Reply:
Yeah, that's the right way to record those events to have them those telemetry events recorded. This can be a follow-up.

> Reading the Nimbus docs a bit more, though, I don't think exposures are the right tool for us, and so maybe this ought to be false?

My understanding is that, we would want to know when a user sees a sponsored or Firefox Suggest recommendation and then have that measured against those `fx_suggest_suggestion_clicked` events. I am not a data engineer though, so making this false works for now and we can always change it later.

Reply:
I changed this to `hasExposure: false` and removed `exposureDescription`, but then when I ran `./gradlew clean` and `./gradlew app:assembleDebug` again, I think it regenerated it. I see all the other features in this file all have `hasExposure: true` and an empty description, too... 🤷🏼‍♀️